src/epcpm/parameterstobitfieldsc.py

```diff
@@ -84,14 +91,25 @@ def sunspec_node_wanted(node: typing.Any) -> bool:
 
             return True
 
-        if self.sunspec_root is None:
-            parameter_uuid_to_sunspec_node = {}
+        if self.sunspec1_root is None:
```

Comment:
Is this one of those instances where it actually needs to be `is None` vs `if not self.sunspec1_root`?

Reply:
Disregard for now, can revisit later